tests/src/test/java/alluxio/LocalAlluxioClusterResource.java

```diff
@@ -127,42 +126,48 @@ public LocalAlluxioCluster get() {
     return mLocalAlluxioCluster;
   }
 
-  private void applyConfParams() throws IOException {
-    mLocalAlluxioCluster.initializeTestConfiguration();
-    // Override the configuration parameters with mConfParams
-    for (int i = 0; i < mConfParams.length; i += 2) {
-      Configuration.set(mConfParams[i], mConfParams[i + 1]);
-    }
+  public LocalAlluxioClusterResource setProperty(PropertyKey key, Object value) {
+    mConfKeys.add(key);
+    mConfValues.add(value.toString());
+    return this;
+  }
+
+  /**
+   * @return whether the local cluster has been started
+   */
+  public boolean isStarted() {
+    return mStartCluster;
```

Comment:
we should use separate variables for tracking whether the cluster is supposed to be started and whether the cluster has been started